book/src/specs/kimchi.md

```diff
@@ -1983,6 +1983,18 @@ pub struct ProofEvaluations<Evals> {
     pub lookup_sorted: [Option<Evals>; 5],
     /// evaluation of runtime lookup table polynomial
     pub runtime_lookup_table: Option<Evals>,
+
```

Comment:
As said in a previous PR, the diagram has to be updated according to this change